lib/containers/docker.pm

```diff
@@ -26,12 +26,16 @@ sub configure_insecure_registries {
     my $registry = registry_url();
     # The debug output is messing with terminal in migration tests
     my $debug = (get_var('UPGRADE')) ? 'false' : 'true';
-    # Allow our internal 'insecure' registry
-    assert_script_run(
-'echo "{ \"debug\": ' . $debug . ', \"insecure-registries\" : [\"localhost:5000\", \"registry.suse.de\", \"' . $registry . '\"] }" > /etc/docker/daemon.json');
-    assert_script_run('cat /etc/docker/daemon.json');
+    # Allow our internal 'insecure' registry only if REGISTRY variable is set
+    my $str;
+    if (get_var('REGISTRY')) {
```

Comment:
```suggestion
    if ($registry) {
```

Reply:
No, because this condition will be always true.. If `REGISTRY` var is not present, `$registry` will be `docker.io` returned from `my $registry = registry_url();` and in this case, we don't want to

Reply:
OK, resolved.